src/Sarif/ExtensionMethods.cs

```diff
@@ -297,7 +298,14 @@ public static string GetFirstSentence(string text)
 
                     case '.':
                     {
-                        if (withinQuotes || withinParentheses) { continue; }
+                        if (withinQuotes || withinParentheses || withinEllipsis) { continue; }
+                        if (length < text.Length && text[length] == '.')
```

Comment:
This code would regard a series of four dots as an ellipsis. But I'm not sure the situation warrants any additional complexity.